documentacion/posintegrado/README.md

```diff
@@ -12,18 +12,27 @@ También puede probar las DLLs que se adjuntan en el [último release](https://g
 
 Esta librería y sus dependencias son requisitos para utilizar el SDK.
 
-Por el momento, el SDK está disponible para [.Net](https://github.com/TransbankDevelopers/transbank-pos-sdk-dotnet) y lo puedes encontrar en [NuGet.org](https://www.nuget.org/packages/TransbankPosSDK/) para instalarlo puedes utilizar por ejemplo el package manager de VisualStudio.
+Por el momento, el SDK está disponible para [.NET](https://github.com/TransbankDevelopers/transbank-pos-sdk-dotnet) y [Java](https://github.com/TransbankDevelopers/transbank-pos-sdk-java). Para .NET lo puedes encontrar en [NuGet.org](https://www.nuget.org/packages/TransbankPosSDK/) para instalarlo puedes utilizar por ejemplo el package manager de VisualStudio.
```

Comment:
is there a maven URL that we can share in here?

Reply:
?

Reply:
This URL goes to Github. In the next section is the URL to Maven (which doesn't exist right now)